### PR TITLE
Coding standard fixes

### PR DESCRIPTION
**Pull request by vijaycs85**
Please merge after #5

### CODE REVIEW

File: src/ApplenewsTemplateInterface.php [hunk 1]
```diff
@@ -4,13 +4,16 @@
 
 use Drupal\Core\Config\Entity\ConfigEntityInterface;
 
+/**
+ *
```
Comment (juampynr) [inline src/ApplenewsTemplateInterface.php]: Shall we take the chance to document this interface?

File: src/ApplenewsTemplateSelection.php [hunk 2]
```diff
@@ -5,18 +5,21 @@
 use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\StringTranslation\StringTranslationTrait;
 
+/**
+ *
```
Comment (juampynr) [inline src/ApplenewsTemplateSelection.php]: Same here.

File: src/Controller/ApplenewsPreviewController.php [hunk 3]
```diff
@@ -6,11 +6,15 @@
 use Drupal\Core\Link;
 use Drupal\Core\Url;
 use Drupal\node\NodeInterface;
-use Symfony\Component\HttpFoundation\BinaryFileResponse;
-use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 
+/**
+ *
```
Comment (juampynr) [inline src/Controller/ApplenewsPreviewController.php]: Please document me :smile_cat:

File: src/Controller/ApplenewsPreviewController.php [hunk 4]
```diff
@@ -6,11 +6,15 @@
...
+ */
 class ApplenewsPreviewController extends ControllerBase {
 
+  /**
+   *
```
Comment (juampynr) [inline src/Controller/ApplenewsPreviewController.php]: This one, and the following ones, too.

File: src/Form/TextStyleForm.php [hunk 5]
```diff
@@ -235,46 +235,44 @@ public function form(array $form, FormStateInterface $form_state) {
...
       '#default_value' => '',
-      '#description' => $this->t('')
+      '#description' => $this->t(''),
     ];
     $form['lineHeight'] = [
       '#type' => 'textfield',
       '#title' => $this->t(''),
       '#default_value' => '',
-      '#description' => $this->t('')
+      '#description' => $this->t(''),
```
Comment (juampynr) [inline src/Form/TextStyleForm.php]: Unrelated: so many fields with empty titles and descriptions. Is this intended?

**Review by juampynr — CHANGES_REQUESTED** [review]

File: src/ApplenewsTemplateInterface.php (hunk 1 above)
Comment (vijaycs85) [reply src/ApplenewsTemplateInterface.php]: Fixed.

File: src/ApplenewsTemplateSelection.php (hunk 2 above)
Comment (vijaycs85) [reply src/ApplenewsTemplateSelection.php]: done.

File: src/Controller/ApplenewsPreviewController.php (hunk 3 above)
Comment (vijaycs85) [reply src/Controller/ApplenewsPreviewController.php]: done.

File: src/Controller/ApplenewsPreviewController.php (hunk 4 above)
Comment (vijaycs85) [reply src/Controller/ApplenewsPreviewController.php]: @juampynr killed all the occurrences of `\/\*\*\n\s\*\n` in Directory modules/contrib/applenews 👍

File: src/Form/TextStyleForm.php (hunk 5 above)
Comment (vijaycs85) [reply src/Form/TextStyleForm.php]: Right, we need them for component styling. we don't use them yet. but I can add a title for now.

**Comment by vijaycs85** [comment]
Thanks for the review @juampynr. I have addressed them all.

File: src/Form/TextStyleForm.php [hunk 6]
```diff
@@ -235,46 +235,44 @@ public function form(array $form, FormStateInterface $form_state) {
         'justified' => 'Justified',
       ],
       '#default_value' => '',
-      '#description' => $this->t('')
+      '#description' => $this->t(''),
     ];
     $form['lineHeight'] = [
       '#type' => 'textfield',
-      '#title' => $this->t(''),
+      '#title' => $this->t('Line height'),
```
Comment (juampynr) [inline src/Form/TextStyleForm.php:242]: :+1:

File: src/Plugin/ApplenewsComponentTypeBase.php [hunk 7]
```diff
@@ -171,11 +168,17 @@ public function getComponentType() {
     return $this->pluginDefinition['component_type'];
   }
 
+  /**
+   *
```
Comment (juampynr) [inline src/Plugin/ApplenewsComponentTypeBase.php:172]: There are still a few like this one but this is already a huge improvement in coding standards so let's move one and merge it.

Comment (vijaycs85) [reply src/Plugin/ApplenewsComponentTypeBase.php:172]: sorry about that @juampynr. Looks like my regex was wrong or didn't cover all cases. Issued follow up fixes at #7

**Pull request merged**